### ISSUE TITLE
Is there a way to detect AltGr is being detected

### ISSUE THREAD

**Issue by 1omarsaid** [jaywcjlove/hotkeys#231]
Hello Team, 

We are currently using this library in our application for keyboard shortcuts, however, we currently have an issue since we use internalization in our app and some of the keys conflict with polish special characters. Here is an example, in the application, we set it so that "Alt +Ctrl + C" goes to the chat section of the app, where also the "right Alt + C" is used for "ç" special characters also triggers the "Alt + Ctrl + C" keyboard shortcut so the special character is never presented. This is occurring for all of our new languages.

In short, there should be a way to detect if "AltGr" is being pressed.

**Comment by MediaMemphis** [comment]
In my case I try to catch altgr+r, but it fits to the title/topic
